src/frontend/components/CreatePlaylistDialog/CreatePlaylistDialog.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+
+import Dialog from "@mui/material/Dialog";
+import DialogTitle from "@mui/material/DialogTitle";
+import TextField from "@mui/material/TextField";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import { useAuth, useUser } from "../../context";
+import { addNewPlaylistService } from "../../services";
+
+function CreatePlaylistDialog({ onClose, open }) {
+  const [playlistName, setPlaylistName] = useState("");
+
+  const {
+    auth: { token },
+  } = useAuth();
+
+  const { userDispatch } = useUser();
+
+  const handleClose = () => {
+    onClose();
+  };
+
+  return (
+    <Dialog onClose={handleClose} open={open}>
+      <DialogTitle sx={{ backgroundColor: "primary.main" }}>
+        Create New Playlist
+      </DialogTitle>
+
+      <Box
+        sx={{
+          bgcolor: "primary.medium",
+          display: "flex",
+          flexDirection: "column",
+        }}
+      >
+        <TextField
+          label="Enter Playlist Name"
+          variant="filled"
+          color="secondary"
+          sx={{
+            bgcolor: "primary.faint",
+            m: 1,
+          }}
+          value={playlistName}
+          onChange={e => setPlaylistName(e.target.value)}
+        />
+        <Button
```

Comment:
Personal opinion - Have the background color of text create playlist & the button div same & can give text input a different shade